src/pug/includes/body.pug

```diff
@@ -3,24 +3,30 @@ include ./fork.pug
 .container
 
 	section.main.flex
-
-		.heading.flex
-			h1 devSwag
-			p 😎 swag opportunities for developers
-			p As a developer, nothing piques my interest more than developer swag! I created devSwag to scratch my own itch by starting a curation of such opportunities in Feb '18. These are crowdsourced and verified by the community. All of them ship worldwide!
+		.flex.box.banner
+			p Want your company's swag to be listed / unlisted? Want to advertise? #[a.btn.secondary(href="https://t.me/swapagarwal") Contact us]
+		.flex.box.hero
+			.heading.flex
+				h1.logo
+					img(src="/assets/img/devswag-logo.svg", alt="DevSwag")
+					span.alt-logo DevSwag
+				h2 swag opportunities for developers 😎
+				p As a developer, nothing piques my interest more than developer swag!
+					br
```

Comment:
Not sure a our that alignment, can you check?

Reply:
@anhek